src/ocean/interfaces/Compute.ts

```diff
@@ -52,4 +53,5 @@ export interface ComputeAlgorithm {
   transferTxId?: string
   dataToken?: string
   algoCustomParameters?: { [key: string]: any }
+  userCustomParameters?: UserCustomParameters
```

Comment:
shouldn't userCustomParameters have the same type/structure of { [key: string]: any }, which is the key-value pair

Reply:
it is.  it's imported from provider.ts at the beginning of the code

Reply:
you are right, mistaken that to another var of similar naming